snapshots/sample/app/src/main/kotlin/com/emergetools/snapshots/sample/ui/TextRowWithIcon.kt

```diff
@@ -25,15 +26,17 @@ fun TextRowWithIcon(
 
 @Preview
 @FontScalePreviews
+@LocalePreviews
```

Comment:
Ran locally and this outputted 5 previews, matching what I expected. Awesome job!

Might be good for us to add a test case in `snapshots-processor-test` for this, but inherently that's a bit tough as it currently relies on a single `/input` directory producing outputs. Maybe adding an `input2` or `submodule` containing a multipreview with an import/usage in `input` would produce a valid test.

Reply:
Yea testing this has proven to be difficult, I'll keep trying some things.